aat/main_test.go

```diff
@@ -51,12 +46,41 @@ func TestMain(m *testing.M) {
 	//router.DebugEnabled = true
 	router.SetLogger(rtrLogger)
 
+	crAuth, err := auth.NewCRAuthenticator(&testCRAuthenticator{})
+	if err != nil {
+		panic(err)
+	}
+
 	// Create router instance.
-	nxr = router.NewRouter(autoRealm, strictURI)
-	nxr.AddRealm(wamp.URI(testRealm), anonAuth, allowDisclose)
+	routerConfig := &router.RouterConfig{
+		RealmConfigs: []*router.RealmConfig{
+			&router.RealmConfig{
+				URI:           wamp.URI(testRealm),
+				StrictURI:     false,
+				AnonymousAuth: true,
+				AllowDisclose: false,
+				Broker:        true,
+				Dealer:        true,
```

Comment:
I think broker and dealer should be enabled by default, if we even want to give an option to disable them (since the only overhead is a an idle goroutine and small struct).  If we even have the options then should change to: `DisableBroker` and `DisableDealer`  or something similar (`NoBroker`, `BrokerOff`, etc)